jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -31,6 +31,8 @@ class LineListTool(TemplateMixin):
     rs_slider_half_range = Float(0.1).tag(sync=True)
     rs_slider_step_auto = Bool(True).tag(sync=True)
     rs_slider_step = Float(0.01).tag(sync=True)
+    rs_redshift_step = Float(1).tag(sync=True)
+    rs_slider_ndigits = Int(1).tag(sync=True)
```

Comment:
Should we start using `IntHandleEmpty` and `FloatHandleEmpty` from #1041 or is that not applicable here?

Reply:
that shouldn't be applicable here since these traitlets are not exposed as inputs to the user and are only set internally or through the API.